style.css

```diff
@@ -0,0 +1,391 @@
+* {
+	box-sizing: border-box;
+}
+
+
+html {
+	font-size: 62.5%;
+	scroll-behavior: smooth;
+}
+
+
+@font-face {
+	font-family: walsheim;
+	src: url(GTWalsheimPro-Light.ttf);
+}
+
+
+body {
+	font-family: walsheim;
+}
+
+
+nav {
+	position: fixed;
+	width: 100%;
+	padding:20px;
+	display: flex;
+	justify-content: space-between;
+	background-color: #fef5ec;
+	font-size: 2rem;
+}
+ 
+ 
+nav a {
+	padding: 10px;
+	color: #002d5d;
+	text-decoration: none;
+}
+
+
+.navLinks a:hover { 
+	color: #2eccd3;
+}
+
+
+.logo {
+	color: #002d5d;
+}
+
+
+header {
+	background-image: url('Header.jpg');
+	background-size: cover;
+	height: 800px;
+	display: flex;
+	justify-content: center;
+}
+
+
+header .intro h1 {
+	font-weight: 900;
+	margin: 0 auto;
+	font-size: 5rem;
+}
+
+header .intro {
+	text-align: center;
+	padding-top: 350px;
+	font-size: 2rem;
+}
+
+.about {
+	display: flex;
+	padding: 80px 140px;
+	background-color: #f69f81;
+}
+
+.profileImage h2 {
+	font-size: 3rem;
+	margin: 0;
+	padding-bottom: 20px;
+	
+}
+
+.profileImage img {
+	width: 300px;
+}
+
+section p {
+	font-size: 2rem;
+	padding: 120px 0px 80px 80px;
+}
+
+main {
+	display: flex;
+	padding: 0px 120px 80px 120px;
+	background-color: #fef5ec;
+}
+
+
+.projectsIntro {
+	padding: 20px 140px;
+	background-color: #fef5ec;
+}
+
+.projectsIntro h2 {
+	margin: 0;
+	padding: 40px 0px 0px 0px;
+	font-size: 3rem;
+}
+
+.projectBox {
+	width: 33%;
+	background-color: #c7f4f8;
+	margin-left: 20px;
+	margin-right: 20px;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	padding: 10px 30px 30px 30px;
+}
+
+.projectBox h3 {
+	width: 100%;
+	text-align: center;
+	font-size: 2rem;
+}
+
+.projectBox img {
+	width: 80%;
+}
+
+.projectBox p {
+	font-size: 1.8rem;
+	text-align: center;
+	padding: 20px;
+	margin: 0;
+}
+
+.projectLinks a {
+	margin-right: 20px;
+	text-decoration: none;
+	color: #002d5d;
+	font-size: 1.4rem;
+}
+
+.projectLinks a:hover {
+	color: #2eccd3;
+}
+
+.contact {
+	width: 100%;
```

Comment:
this isnt doing anything as divs are block level elements by default so will take up 100% width without having to specify it